engine/src/main/java/org/terasology/rendering/dag/FinalPostProcessingNode.java

```diff
@@ -143,14 +145,14 @@ private void renderFinalMonoImage() {
     // TODO: have a flag to invert the eyes (Cross Eye 3D), as mentioned in
     // TODO: http://forum.terasology.org/threads/happy-coding.1018/#post-11264
     private void renderFinalStereoImage(RenderingStage renderingStage) {
-        if (postProcessor.isNotTakingScreenshot()) {
-            sceneFinal.bind();
-        } else {
-            ocUndistorted.bind();
-        }
-
         switch (renderingStage) {
             case LEFT_EYE:
+                if (postProcessor.isNotTakingScreenshot()) { // TODO: verify if this works
```

Comment:
I'm an idiot. It's obvious it can't work. The bindings in the IF block at the beginning of the method affect the quad rendering at the beginning of the RIGHT_EYE case. Somehow I missed that line. If it hadn't been there it would have worked. Sorry for the waste of time.